packages/enketo-core/src/widget/draw/draw-widget.js

```diff
@@ -49,7 +49,7 @@ class DrawWidget extends Widget {
                 penColor: that.props.colors[0] || 'black',
             });
 
-            this.pad.addEventListener('change', () => {
+            this.pad.addEventListener('endStroke', () => {
```

Comment:
This was the change in https://github.com/enketo/enketo/commit/4f6abec01a9aae681df8dfb3ef0efc5b3dbda5e4# that caused the regression. It appears that the change event (instead of endStroke) works, but is not supported by signaturepad (https://github.com/szimek/signature_pad?tab=readme-ov-file#events). The problem with using it is that `this.pad.off()` doesn't actually turn it off. I don't know if there was a specific reason for using 'change' though .

Reply:
Ah, no, I looked a little deeper and that doesn't make sense either.

Reply:
Can we just move the event listener below the `if (existingFilename)` block? The problem is that setting the existing image registers as a change, right?

Reply:
Oops! Sorry about that attempt to revert that key fix/feature!  I also looked deeper and also concluded extending `off()` wouldn't work. We could turn it off in draw-widget.js with removeEventListener(). Might have to be done at several places (and turned on again in several places). Or in the `ResizableSignaturePad` by not firing a change event if it's `off` (but not sure if we can easily create an `isOff()` check).

Reply:
I'll revisit this. Probably not until next week though. Thanks for reviewing!